example/lib/main.dart

```diff
@@ -15,8 +15,8 @@ class MyApp extends StatefulWidget {
 }
 
 class _MyAppState extends State<MyApp> {
-  List<String> toPrint = ['trying to connect'];
-  SocketIOManager manager;
+  List<String?> toPrint = ['trying to connect'];
+  SocketIOManager? manager;
```

Comment:
This can be `late SocketIOManager manager`, thus avoiding the need to add `manager!` everywhere else in this file